contracts/FlowIDTableStaking.cdc

```diff
@@ -43,7 +43,6 @@ pub contract FlowIDTableStaking {
     pub event NewNodeCreated(nodeID: String, role: UInt8, amountCommitted: UFix64)
```

Comment:
Doesn't seem like `contracts/FlowIDTableStaking.cdc` is synced up with the latest changes.

Reply:
I synced it up here, but I'm gonna need your help getting the feature branch working with the changes once we get all these merged because if I update this file, the tests don't run because these changes break the flow-go integration. I'll ping you again once we have all these merged to the feature branch so we can get it synced up properly